app/index.html

```diff
@@ -28,7 +28,7 @@
 <link rel="icon" href="img/icons/512.png" sizes="512x512">
 <link rel="icon" href="img/icon.svg" sizes="any" type="image/svg+xml">
 <title>Project Link</title>
-<script defer src="js/initapp.js"></script>
+<script src="js/alameda.js" data-main="js/bootstrap"></script>
```

Comment:
Doesn't it work with defer anymore?
I really wish we can split independent chunks of JavaScript with `defer` and `async`.